mcux/drivers/lpc/fsl_iap.c

```diff
@@ -1,666 +1,584 @@
 /*
- * Copyright 2018-2021 NXP
+ * Copyright 2018-2020 NXP
```

Comment:
True, this goes backward with the `./mcux/scripts/import_mcux_sdk.py`. Should I manually add only the device for LPC51U68?

Reply:
Yes. Can you note that in your commit message please?

Reply:
@parthitce, I get that the script will cause this to go backwards if you grab an older SDK for this platform but you might find that the newer driver will work with your platform. Can you test for this to see if it actually needs to be moved backwards? If it does then we may need to put this file under the device instead of in the generic mcux/drivers/lpc directory

Reply:
@dleach02 I was pulling the latest SDK. let me retry with fresh download again. Looks like update with SDK 2.10.0  moving backward.

Reply:
@parthitce, what probably isn't obvious is that NXP releases SDKs targeted at SOC platforms but the IP blocks are shared across a bunch of our chips. What can happen is that the driver can be updated by the SDK for one SOC while another SOC may not have an updated SDK... but the driver update still applies. So we want to make sure we don't go backward on any driver that already exists in the HAL. When I was doing the recent update of the 2.10 SDKs I was doing one platform at a time and would check to see if it touched any of the drivers.